packages/api/src/modules/user/user.service.ts

```diff
@@ -311,6 +323,116 @@ export class UserService {
         return user;
     }
 
+    sanitizeActivationUserInfo(userInfo) {
+        return Object.keys(userInfo).reduce((acc, key) => {
+            const prop = userInfo[key];
+            if (key === "jobType") acc[key] = prop.map(type => sanitizeToPlainText(type?.toString()));
```

Comment:
Ça va retourner un array de string. Si c'est pour stocker en base je comprend pas pourquoi on garde pas l'enum et si c'est pour transmettre à brevo ça ne sera pas accepté

Reply:
surtout que si j'ai bien compris c'est validé par ailleurs

Reply:
alors effectivement j'avais fait ce dev avant la validation du champ... je vais supprimer l'itération sur l'objet entier et mettre les champs non vérifiés jusque là